.travis.yml

```diff
@@ -3,10 +3,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
 dist: bionic
-language: go
-
-go:
-  - 1.14.4
```

Comment:
If we remove this (which makes sense as long as we do only docker build) then we probably should also remove below `go_import_path`? 

BTW: as this is far away from a change, i'can do clicky suggestion. So seems you can do clicky suggestsions. Seems while you can do them outside of changes, it can only be done at most two lines away from a change?

Reply:
done